plugins/source/azure/resources/services/logic/app_workflows.go

```diff
@@ -1,332 +0,0 @@
-			},
-		},
-	}
-}
-
-// ====================================================================================================================
-//                                               Table Resolver Functions
-// ====================================================================================================================
-
-func fetchLogicAppWorkflows(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
-	svc := meta.(*client.Client).Services().Logic.Workflows
-	var top int32 = 100
-	response, err := svc.ListBySubscription(ctx, &top, "")
-	if err != nil {
-		return diag.WrapError(err)
-	}
-	for response.NotDone() {
-		res <- response.Values()
-		if err = response.NextWithContext(ctx); err != nil {
-			return diag.WrapError(err)
-		}
-	}
-	return nil
-}
-
-func endpointsConfigurationResolver(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
-	workflow := resource.Item.(logic.Workflow)
-	if workflow.EndpointsConfiguration == nil {
-		return nil
-	}
-	b, err := json.Marshal(*workflow.EndpointsConfiguration)
-	if err != nil {
-		return diag.WrapError(err)
-	}
-	return diag.WrapError(resource.Set(c.Name, b))
-}
-func accessControlResolver(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
-	workflow := resource.Item.(logic.Workflow)
-	if workflow.AccessControl == nil {
-		return nil
-	}
-	b, err := json.Marshal(*workflow.AccessControl)
-	if err != nil {
-		return diag.WrapError(err)
-	}
-	return diag.WrapError(resource.Set(c.Name, b))
-}
-
-func definitionResolver(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
-	definition := resource.Item.(logic.Workflow).Definition
-	if definition == nil {
-		definition = make(map[string]interface{})
-	}
-	b, err := json.Marshal(definition)
-	if err != nil {
-		return diag.WrapError(err)
-	}
-	return diag.WrapError(resource.Set(c.Name, b))
-}
-func parametersResolver(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
-	workflow := resource.Item.(logic.Workflow)
-	if workflow.Parameters == nil {
-		return nil
-	}
-	b, err := json.Marshal(workflow.Parameters)
-	if err != nil {
-		return diag.WrapError(err)
-	}
-	return diag.WrapError(resource.Set(c.Name, b))
-}
-
-func identityUserAssignedIdentitiesResolver(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
-	workflow := resource.Item.(logic.Workflow)
-	if workflow.Identity == nil || workflow.Identity.UserAssignedIdentities == nil {
-		return nil
-	}
-	b, err := json.Marshal(workflow.Identity.UserAssignedIdentities)
-	if err != nil {
-		return diag.WrapError(err)
-	}
-	return diag.WrapError(resource.Set(c.Name, b))
-}
-
-func fetchDiagnosticSettingsResolver(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
Extracted to a relation in `plugins/source/azure/resources/services/logic/diagnostic_settings.go`